src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -464,6 +464,18 @@ private static bool TryGetTextForKeyword(SyntaxToken token, [NotNullWhen(true)]
                     text = Keyword("defaultconstraint");
                     return true;
                 }
+
+                if (token.Parent is DefaultSwitchLabelSyntax or GotoStatementSyntax)
+                {
+                    text = Keyword("defaultcase");
```

Comment:
This will redirect to https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/statements/selection-statements

Currently there doesn't seem to be anything about `goto default` there, but I opened https://github.com/dotnet/docs/pull/32196